neqo-transport/src/packet/mod.rs

```diff
@@ -866,10 +866,16 @@ impl Deref for DecryptedPacket {
 #[cfg(test)]
 mod tests {
     use super::*;
-    use crate::crypto::{CryptoDxState, CryptoStates};
+    #[cfg(not(feature = "fuzzing_t"))]
```

Comment:
I think that we can just disable all of these tests.  Far too much of this is not relevant to fuzzing (it's internals) anyway.  We'll get more value from fuzzing the end-to-end testing anyway.  Picking individual pieces out is too hard to maintain.